README.md

```diff
@@ -28,6 +28,11 @@ In that form, they cannot attack anymore, but have a defense bonus.
 
 ### Linux
 
+Install SDL
```

Comment:
This should probably say "SDL2" or "SDL 2", right?

Reply:
Done. Also added instructions for Ubuntu.